pgcli/main.py

```diff
@@ -405,6 +420,19 @@ def get_completions(self, text, cursor_positition):
         return self.completer.get_completions(
             Document(text=text, cursor_position=cursor_positition), None)
 
+    def toggle_expanded_output(self):
+        """Callback for the \\x special command"""
+        self.use_expanded_output = not self.use_expanded_output
+        message = u"Expanded display is "
+        message += u"on." if self.use_expanded_output else u"off."
+        return [(None, None, None, message)]
+
+    def toggle_timing(self):
+        self.timing_enabled = not self.timing_enabled
+        message = "Timing is "
+        message += "on." if self.timing_enabled else "off."
+        return [(None, None, None, message)]
+
```

Comment:
Since we're making PGSpecial as a class, can we maintain these global vars and functions inside that class as member vars and methods? 

Imagine PGSpecial as a standalone library that someone else wants to use in their own REPL (say someone wants to do a GUI equivalent of PGCli). It'll be good to provide as many commands as possible through PGSpecial and only fallback to using the registration hooks when we need special access to connection object (\c) or completion object (\#). 

Thoughts?